.travis.yml

```diff
@@ -2,17 +2,22 @@ sudo: false
 language: node_js
 cache:
   directories:
-    - node_modules
+  - node_modules
```

Comment:
It will work either way. Indentation is preferred though for improved readability.